config/owasp/suppressions.xml

```diff
@@ -18,4 +18,11 @@
         <cve>CVE-2020-29244</cve>
         <cve>CVE-2020-29245</cve>
     </suppress>
+    <suppress until="2021-11-16">
+        <notes><![CDATA[
+   file name: lang-tag-1.4.4.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.nimbusds/lang\-tag@.*$</packageUrl>
```

Comment:
we have already added the suppression for this jar. we just need to add this cve alone - like this https://github.com/hmcts/rd-judicial-api/pull/162/files